.github/workflows/flutter_ci.yml

```diff
@@ -0,0 +1,87 @@
+name: flutter-ci
```

Comment:
I'm not in in favor of a separate CI for both packages. This is largely duplicating the workflow and increasing maintenance effort. Can't we just remove 1 single line as you [suggested](https://github.com/parse-community/Parse-SDK-Flutter/pull/969#issuecomment-1763342748)?

Reply:
Done, please check again

Reply:
Great!